src/components/templates/Audience/Audience.tsx

```diff
@@ -342,16 +344,11 @@ export const Audience: React.FC<AudienceProps> = ({ venue }) => {
       <>
         <div className="emoji-container">
           {burningReactions.map((reaction) => (
-            <button
+            <Reaction
               key={reaction.name}
-              className="reaction"
-              onClick={() => reactionClicked(reaction.type)}
-              id={`send-reaction-${reaction.type}`}
-            >
-              <span role="img" aria-label={reaction.ariaLabel}>
-                {reaction.text}
-              </span>
-            </button>
+              reaction={reaction}
+              reactionClicked={reactionClicked}
```

Comment:
This prop should be named like a callback `onClickReaction` or just `onClick`

Reply:
Also should we disable this while the reaction is still being send? What do you think. For example, we can use `useAsyncFn` and disable this while the request is pending.

Reply:
I'd also change the name of the function that's passed to this prop. `reactionClicked` doesn't really indicate what this function does. I'd rename it to the actual functionality that it does. Ex: `onReactionClick={addReaction}` or `onReactionClick={sendReaction}`

Reply:
@denisdimitrov I'm not sure that it's ok to block users to send reactions
Other stuff is fixed in d4f3cf3bd46debf17b8f6f9452f37bd409104f32

Reply:
Let's keep it always enabled for now, but keep in mind that people can spam emotes atm.